packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -300,15 +298,17 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
     final List<Widget> headerWidgets = <Widget>[];
     double startPadding = 24.0;
     if (_selectedRowCount == 0) {
-      headerWidgets.add(Expanded(child: widget.header));
-      if (widget.header is ButtonBar) {
-        // We adjust the padding when a button bar is present, because the
-        // ButtonBar introduces 2 pixels of outside padding, plus 2 pixels
-        // around each button on each side, and the button itself will have 8
-        // pixels internally on each side, yet we want the left edge of the
-        // inside of the button to line up with the 24.0 left inset.
-        // TODO(ianh): Better magic. See https://github.com/flutter/flutter/issues/4460
-        startPadding = 12.0;
+      if (widget.header != null){
```

Comment:
nit: please be careful with style -- there should be a space between `)` and `{` for example

Reply:
Tnx. I'll correct the style.